src/elm/Components/Face.elm

```diff
@@ -24,9 +24,5 @@ highlightSelectedFace logstage model face =
                 model.newLog.postFace
             )
     in
-        case (faceToInt face) == selectedFace of
-            True ->
-                "face-container b--green"
+        if (faceToInt face) == selectedFace then "face-container b--green" else "face-container b--white"
```

Comment:
Sorry @lucymk could you get all of these using the ifThenElse helper in Helpers.Utils? ;)

Reply:
oooh good idea, will do! 👍